### DISCUSSION TITLE
[Q&A] 使用第三方接口报错的原因？

### DISCUSSION THREAD

**Discussion by hileel** [lobehub/lobehub#734] [General Question] [answered]
### 🧐 问题描述 | Proposed Solution

<img width="1275" alt="image" src="https://github.com/lobehub/lobe-chat/assets/6147589/a0ce2d1c-7075-4f15-8a1a-21c7bbec02ef">

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
加上 `/v1` 试试

[a comment by an automated account is not shown]

**Comment by hileel** [comment]
> > Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿
> 
> Try adding `/v1`

有效～感谢
